hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2113,7 +2116,7 @@ private void startTrashEmptier(Configuration conf) throws IOException {
           (PrivilegedExceptionAction<FileSystem>)
               () -> new TrashOzoneFileSystem(i));
       this.emptier = new Thread(new OzoneTrash(fs, conf, this).
-          getEmptier(), "Trash Emptier");
+          getEmptier(), threadPrefix + "Trash Emptier");
```

Comment:
Nit: it would be nice to get rid of the space in thread names.  Makes cutting specific columns from log by whitespace a lot easier.

Reply:
Done !!
I have removed it for this as well in OzoneManager :- 
```
      final Thread t = new Thread(bgSync::triggerRangerSyncOnce,
          threadPrefix + "Trigger_RangerSync");
```